Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1479 @@
+    excluded_list = argToList(arg=excluded_branches)
+    included_list = argToList(arg=included_branches)
+    response = client.commit_list_request(repo, file_path, page, page_size, excluded_list, included_list)
+    results = check_pagination(client, response, limit)
+    human_readable = []
+    key_list = []
+    for value in results:
+        d = {'Author': value.get('author').get('raw'),
+             'Commit': value.get('hash'),
+             'Message': value.get('message'),
+             'CreatedAt': value.get('date')}
+        human_readable.append(d)
+        key_list.append(value.get('hash'))
+
+    headers = ['Author', 'Commit', 'Message', 'CreatedAt']
+    readable_output = tableToMarkdown(
+        name='The list of commits',
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.Commit',
+        outputs=results,
+        raw_response=results,
+        outputs_key_field=key_list
+    )
+
+
+def file_delete_command(client: Client, args: Dict) -> CommandResults:
+    """ Deletes a file in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful action.
+    """
+    repo = args.get('repo', client.repository)
+    message = args.get('message', '')
+    branch = args.get('branch', '')
+    file_name = args.get('file_name', '')
+    author_name = args.get('author_name', None)
+    author_email = args.get('author_email', None)
+    body = {
+        'message': message,
+        'branch': branch,
+        'files': file_name
+    }
+    if author_name and author_email:
+        body['author'] = f'{author_name} <{author_email}>'
+    elif author_name and not author_email:
+        raise Exception('Please enter an email as well.')
+    elif not author_name and author_email:
+        raise Exception('Please enter a name as well.')
+    client.file_delete_request(body, repo)
+    return CommandResults(readable_output='The file was deleted successfully.')
+
+
+def raw_file_get_command(client: Client, args: Dict) -> List[CommandResults]:
+    """ Returns the content of a given file.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A list containing a CommandResult object and a fileResult object.
+    """
+    repo = args.get('repo', client.repository)
+    file_path = args.get('file_path', '')
+    branch = args.get('branch', None)
+    if branch:
+        including_list = [branch]
+    else:
+        including_list = []
+    try:
+        commit_list = client.commit_list_request(repo=repo, file_path=file_path, page=1, page_size=100,
+                                                 excluded_list=[], included_list=including_list)
+    except DemistoException as e:
+        message_arr = e.message.split('\n')
```

Comment:
You're using it several times in the code.
Let's make a helper function with it instead and call it in the commands

Reply:
I use it 4 time in total. 
* Twice with just 1 line: raise Exception(f'The command pull-request-comment-delete failed. {e.message}') 
* Twice with message_arr and m_json but then there are different parts from m_json that are in the exception method. 
Do you think that in this cases I still need to make a helper function?

Reply:
Yeah, I think specifically in this case, it will make the code cleaner and easier to read